src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
+  const rentalConst = 40;
```

Comment:
p2(let)

Reply:
"sum" it's a value thats will change, so I can't use const here. Or u mean something else?

Reply:
I mean that we can do it without let, but probably I though wrong about this. Lets see what menors say)